crowbar_framework/app/models/crowbar_validator.rb

```diff
@@ -101,7 +101,8 @@ def validate_hook(value, rule, path, errors)
       when "IpAddress"
         cidr = (1..128).cover? Integer(value) rescue false
         begin
-          IPAddr.new(value) if !(path.include?("broadcast") && value.empty?) && !cidr
+          IPAddr.new(value) if !(path.is_a?(String) && path.include?("broadcast") \
+                                 && value.empty?) && !cidr
```

Comment:
This is getting unwieldy, can we restructure this to 1) get rid of the \ and 2) not have 4 conditions, 3 of which are nested.